README.md

```diff
@@ -53,39 +154,47 @@ UsageAlert := SUBSCRIBE SELECT * FROM UserTokens WHERE tokens > 100000;
 Create a file `usertokens.sqrl` with the content above and run it with:
 
 ```bash
-docker run -it --rm -p 8888:8888 -p 8081:8081 -p 9092:9092 -v $PWD:/build datasqrl/cmd:latest run usertokens.sqrl
+docker run -it --rm -p 8888:8888 -p 8081:8081 -v $PWD:/build datasqrl/cmd run usertokens.sqrl
```

Comment:
Add tag -> `datasqrl/cmd:0.7.0`? Same for the `docker run` calls after this as well.